From 42a9e17601728a5674ab4ebdfc2b22ce33c0a946 Mon Sep 17 00:00:00 2001
From: Jan Kratochvil <jan.kratochvil@redhat.com>
Date: Wed, 9 May 2012 18:15:58 +0000
Subject: [PATCH] gdb/doc/
 	* gdb.texinfo (Auto-loading): Wrap too long lines in @smallexample.
 	Twice.

---
 gdb/doc/ChangeLog   |    5 +++++
 gdb/doc/gdb.texinfo |    6 ++++--
 2 files changed, 9 insertions(+), 2 deletions(-)

Index: gdb-7.2/gdb/doc/gdb.texinfo
===================================================================
--- gdb-7.2.orig/gdb/doc/gdb.texinfo
+++ gdb-7.2/gdb/doc/gdb.texinfo
@@ -19765,7 +19765,8 @@ or disabled.
 (gdb) show auto-load
 gdb-scripts:  Auto-loading of canned sequences of commands scripts is on.
 libthread-db:  Auto-loading of inferior specific libthread_db is on.
-local-gdbinit:  Auto-loading of .gdbinit script from current directory is on.
+local-gdbinit:  Auto-loading of .gdbinit script from current directory
+                is on.
 python-scripts:  Auto-loading of Python scripts is on.
 safe-path:  List of directories from which it is safe to auto-load files
             is /usr/local.
@@ -19783,7 +19784,8 @@ gdb-scripts:
 Loaded  Script
 Yes     /home/user/gdb/gdb-gdb.gdb
 libthread-db:  No auto-loaded libthread-db.
-local-gdbinit:  Local .gdbinit file "/home/user/gdb/.gdbinit" has been loaded.
+local-gdbinit:  Local .gdbinit file "/home/user/gdb/.gdbinit" has been
+                loaded.
 python-scripts:
 Loaded  Script
 Yes     /home/user/gdb/gdb-gdb.py
